.github/workflows/beaker.yml

```diff
@@ -19,9 +19,17 @@ on:
         default: 45
         required: false
         type: number
+      working-directory:
+        description: The working directory where all jobs should be executed. useful if you've a module in a subdirectory (like a monorepo or a controlrepository)
```

Comment:
I sort of implied to update this and the other instance as well to keep them in sync.

Reply:
yes I want to update them, but Ibak currently only on my phone :D will do it later

Reply:
should be fine now.